reference/library/src/morpheus-master/sass/access.scss

```diff
@@ -0,0 +1,136 @@
+/* access.scss : Styles for access servlet (/access/) directory listings */
+@import "defaults";
+
+@import "base/extendables";
+@import "base/defaults";
+
+.directoryIndex {
+    margin-top: 2em;
+    padding: 1em;
+
+    a {
+	    text-decoration: none !important;
+    }
+
+    .textPanel {
+        font-size: .95em;
+        line-height: 150%;
+        padding:5px 5px  5px  21px;
+    }
+
+    ul {
+        list-style: none;
+        margin:.5em 0 0 0;
+        padding:0;
+
+        li {
+            list-style: none;
+            vertical-align: middle;
+            padding:.3em 0;
+            margin:0;
+
+            img {
+                margin:0 5px 0  0;
+                vertical-align: top;
+                border:none;
+            }
+
+            a {
+	            font-weight:bold;
+            }
+        }
+
+        li.folder {
+            padding:3px;
+
+            a {
+                padding-left:21px;
+                background: url(../../image/silk/folder.png) center left no-repeat !important
+            }
+
+            a:hover {
+                padding-left:21px;
+                background: url(../../image/silk/folder_go.png) center left no-repeat !important
+            }
+        }
+
+        li.upfolder{
+            position:absolute;
+            top:2px;
+            left:2px;
+            margin-bottom:2em;
+
+            a {
+                border:1px solid #ccc;
+                padding:3px;
+                display: inline-block;
+            }
+
+            a:hover {
```

Comment:
Just for the record, you can do this to this way:

     a {
                border:1px solid #ccc;
                padding:3px;
               display: inline-block;
               &:hover{
                          padding-left:21px;
                          background: url(../../image/silk/folder.png) center left no-repeat !important
               }
    } 

Don't repeat this, of course, it's just to say.